examples/views/wait_for_button_press.py

```diff
@@ -0,0 +1,53 @@
+import asyncio
+
+import discord
+from discord.ext import commands
```

Comment:
The import for `discord.ext.commands` is not needed.

Reply:
They're currently using `commands.Bot` for whatever reason